src/app/api/auth/[...nextauth]/route.ts

```
@@ -0,0 +1,65 @@
+import { UserService } from '@/entities/user/model/service'
+import moment from 'moment'
+import NextAuth, { NextAuthOptions, Session, User } from 'next-auth'
+import { JWT } from 'next-auth/jwt'
+import GoogleProvider from 'next-auth/providers/google'
+
+export const authOptions: NextAuthOptions = {
+  providers: [
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
```

Could you throw error if GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET are undefined?